### PR TITLE
Update README for v3.6, etc.

### PR DESCRIPTION
**Pull request by adrianeboyd**
## Description

* Update most recent release
* Switch from azure to GHA CI tests badge
* Remove link to survey
* Format

### Types of change

Docs.

## Checklist

- [x] I confirm that I have the right to submit this contribution under the project's MIT license.
- [ ] I ran the tests, and all new and existing tests passed.
- [x] My changes don't require a change to the documentation, or if they do, I've added all required information.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,23 +6,20 @@ spaCy is a library for **advanced Natural Language Processing** in Python and
...
 
-💥 **We'd love to hear more about your experience with spaCy!**
-[Fill out our survey here.](https://form.typeform.com/to/aMel9q9f)
-
-💫 **Version 3.5 out now!**
+💫 **Version 3.6 out now!**
 [Check out the release notes here.](https://github.com/explosion/spaCy/releases)
 
-[![Azure Pipelines](https://img.shields.io/azure-devops/build/explosion-ai/public/8/master.svg?logo=azure-pipelines&style=flat-square&label=build)](https://dev.azure.com/explosion-ai/public/_build?definitionId=8)
+[![tests](https://github.com/explosion/spaCy/actions/workflows/tests.yml/badge.svg)](https://github.com/explosion/spaCy/actions/workflows/tests.yml)
```
Comment (rmitsch) [inline README.md:22]:
```suggestion
[![Tests](https://github.com/explosion/spaCy/actions/workflows/tests.yml/badge.svg)](https://github.com/explosion/spaCy/actions/workflows/tests.yml)
```

Comment (adrianeboyd) [reply README.md:22]: This is auto-generated and the way the badge works, this doesn't actually change the appearance. (https://github.com/adrianeboyd/spaCy/tree/test/readme-1)

Comment (adrianeboyd) [reply README.md:22]: It looks like you can set it using the name in the workflow file. I'm not sure I want to go change them all?

File: README.md [hunk 2]
```diff
@@ -35,30 +32,30 @@ open-source software, released under the [MIT license](https://github.com/explos
...
+| <a href="https://explosion.ai/spacy-tailored-analysis"><img src="https://user-images.githubusercontent.com/1019791/206151300-b00cd189-e503-4797-aa1e-1bb6344062c5.png" width="125" alt="spaCy Tailored Pipelines"/></a>   | Bespoke advice for problem solving, strategy and analysis for applied NLP projects. Services include data strategy, code reviews, pipeline design and annotation coaching. Curious? Fill in our 5-minute questionnaire to tell us what you need and we'll be in touch! **[Learn more &rarr;](https://explosion.ai/spacy-tailored-analysis)** |
 
 [spacy 101]: https://spacy.io/usage/spacy-101
 [new in v3.0]: https://spacy.io/usage/v3
 [usage guides]: https://spacy.io/usage/
 [api reference]: https://spacy.io/api/
 [models]: https://spacy.io/models
 [universe]: https://spacy.io/universe
-[spaCy VS Code Extension]: https://github.com/explosion/spacy-vscode
+[spacy vs code extension]: https://github.com/explosion/spacy-vscode
```
Comment (svlandeg) [inline README.md:58]: Why lowercase this? "VS Code" in particular looks weird to me lower-cased.

**Pull request merged**